addon/mixins/table-header.js

```diff
@@ -107,6 +107,17 @@ export default Mixin.create({
   iconDescending: '',
 
   /**
+   * See `iconSortable, iconAsending, or iconDescending.  Custom sorting component
```

Comment:
I would change the order
```
Custom sorting component name to use instead of the default `<i class="lt-sort-icon"></i>` template. 
See `iconSortable`, `iconAsending`, or `iconDescending` for more info.
```
> Uses the same properties.

It's unclear what you mean by this sure what you mean by this